src/ostree.cc

```diff
@@ -19,29 +19,24 @@ OstreePackage::OstreePackage(const std::string &ref_name_in, const std::string &
                              const std::string &treehub_in)
     : PackageInterface(ref_name_in, refhash_in, treehub_in) {}
 
-data::InstallOutcome OstreePackage::install(const data::PackageManagerCredentials &cred,
-                                            const PackageConfig &config) const {
-  const char remote[] = "aktualizr-remote";
-  const char *const commit_ids[] = {refhash.c_str()};
-  const char *opt_osname = NULL;
+data::InstallOutcome OstreeManager::pull(const Config &config, const data::PackageManagerCredentials &cred,
+                                         const std::string &refhash) {
   OstreeRepo *repo = NULL;
+  const char *const commit_ids[] = {refhash.c_str()};
   GCancellable *cancellable = NULL;
   GError *error = NULL;
-  char *revision;
   GVariantBuilder builder;
   GVariant *options;
 
-  if (config.os.size()) {
-    opt_osname = config.os.c_str();
-  }
-  boost::shared_ptr<OstreeSysroot> sysroot = OstreeManager::LoadSysroot(config.sysroot);
+  boost::shared_ptr<OstreeSysroot> sysroot = OstreeManager::LoadSysroot(config.pacman.sysroot);
   if (!ostree_sysroot_get_repo(sysroot.get(), &repo, cancellable, &error)) {
-    LOG_ERROR << "Could not get OSTree repo";
+    LOG_ERROR << "could not get repo";
```

Comment:
Not a big deal, but any reason to prefer the less specific error message?

Reply:
Hm, maybe I reverted old message